### ISSUE TITLE
softmax  error

### ISSUE THREAD

**Issue by wangjing60755** [azeme1/keras2ncnn#11]
![image](https://user-images.githubusercontent.com/35362513/108490821-16073300-729b-11eb-8ddb-33762462497d.png)

@azeme1

**Comment by wangjing60755** [comment]
[weights.zip](https://github.com/azeme1/keras2ncnn/files/6009205/weights.zip)

segment model here!

**Comment by wangjing60755** [comment]
model output：

`    output = layers.Conv2D(filters=numclass, kernel_size=(
        1, 1), padding='same', activation='softmax', name='final_layer')(x)`

**Comment by azeme1** [comment]
The reason of the issue. NCNN does not support the merged Softmax activation. I have to write the split rule to tranrform graph to convertible one. If it is urgent you can modify the model 
output = layers.Conv2D(filters=numclass, kernel_size=( 1, 1), padding='same', activation=None, name='final_layer')(x)
output = Softmax(output )

**Comment by azeme1** [comment]
I have fixed the issue. Could you try with new 'main' branch?
To import you model you should modify 
from tensorflow.keras.losses import mae
model_list = [load_model('weights.h5', custom_objects={'bce_dice_loss': mae,
                                                                                      'IOU': mae})]  #issue 11

**Comment by azeme1** [comment]
Succeed to fix the mode and convert it into the NCNN.

https://www.dropbox.com/sh/8anok3k3jxjj81i/AADWMLad_V0MKs4ySN2mgPPda?dl=0
The conversion result can be found here
model_zoo\variouse\issue_00011

**Comment by wangjing60755** [comment]
This model，can you help? train use keras in tf2.0

[Uploading weights_v2.zip…]()

**Comment by wangjing60755** [comment]
[weights_v2.zip](https://github.com/azeme1/keras2ncnn/files/6015349/weights_v2.zip)

this model,can  you help?
train  use  keras in tf2.0.

**Comment by azeme1** [comment]
TensorFlowOpLayer what is inside this layer?
Could you share the actual model code?

**Comment by azeme1** [comment]
Is this actual one?

**Comment by azeme1** [comment]
Is the problem solved?
